style.css

```diff
@@ -0,0 +1,4 @@
+/* styles for the project */
+.fav-color {
```

Comment:
- [ ] You do not have any class with the name `fav-color` in your HTML code and this has resulted in the color `#2d2d91` not been implemented. Please fix this. If this isn't clear, you can find more Information [here](https://www.w3schools.com/html/html_classes.asp).

Reply:
- This is quite enlightening. 🤝